ddl/job_table.go

```diff
@@ -225,18 +228,42 @@ func (d *ddl) delivery2worker(wk *worker, pool *workerPool, job *model.Job) {
 			asyncNotify(d.ddlJobCh)
 			metrics.DDLRunningJobCount.WithLabelValues(pool.tp().String()).Dec()
 		}()
-		// we should wait 2 * d.lease time to guarantee all TiDB server have finished the schema change.
-		// see waitSchemaSynced for more details.
+		// check if this ddl job is synced to all servers.
 		if !d.isSynced(job) || d.once.Load() {
-			err := wk.waitSchemaSynced(d.ddlCtx, job, 2*d.lease)
-			if err == nil {
-				d.once.Store(false)
+			if variable.EnableMDL.Load() {
+				exist, err := checkMDLInfo(job.ID, d.sessPool)
```

Comment:
if this scenario will occur?
line 285 just cleanMDLInfo, DDL owner crashed,
another DDL owner is setup, in 234 line the MDL not exist, it will in waitSchemaSynced status and can not restor back?

Reply:
If MDL does not exist, it won't in waitSchemaSynced